### PR TITLE
[*] Command Generate - Rename model `sessions` to `session` to avoid conflict with forest routes

### PR DESCRIPTION
**Pull request by VincentMolinie**
# How to reproduce
Create a model named `sessions`.
Create a new project with lumber.
Try to login to your liana.

# What has been done ?
I just rename the model to `session` without the s to avoid conflict with the `POST /forest/sessions`

### CODE REVIEW

**Comment by VincentMolinie** [comment]
[(?) P1 - Users can't log in if they have a model called sessions](https://trello.com/c/zSj11msK/3935-p1-users-cant-log-in-if-they-have-a-model-called-sessions)

File: services/dumper.js [hunk 1]
```diff
@@ -76,6 +76,13 @@ function Dumper(config) {
     return _.kebabCase(table);
   }
 
+  function getModelName(table) {
+    if (table === 'sessions') {
+      return 'session';
+    }
```
Comment (arnaudbesnier) [inline services/dumper.js]: It does not look to be a long term solution against this kind of route conflicts issue.

File: services/dumper.js [hunk 2]
```diff
@@ -76,6 +76,13 @@ function Dumper(config) {
     return _.kebabCase(table);
   }
 
+  function getModelName(table) {
+    if (table === 'sessions') {
```
Comment (larcin) [inline services/dumper.js]: [Notice] You should add a comment here to explain why thus edge case needs to be handled

File: services/dumper.js (hunk 1 above)
Comment (VincentMolinie) [reply services/dumper.js]: What solution do you suggest ?

**Pull request closed without merging**